specification/dartLangSpec.tex

```diff
@@ -36,6 +38,8 @@
 %   Function.
 % - Generalize specification of type aliases such that they can denote any
 %   type, not just function types.
+% - Added requirement that the iterator of a for-in statement must have
```

Comment:
Move this to 2.3 and change version to 2.3

Reply:
Done